desktop/apps/articles/templates/section.jade

```diff
@@ -23,3 +23,6 @@ block body
         hr
         +share(section.get('title') + ' on Artsy')
   include full_feed
+
+  if veniceBanner != undefined
```

Comment:
Pro tip: You can rely on ["falseyness"](http://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript) here and just use `if veniceBanner`.